department-of-veterans-affairs/va.gov-cms
# VACMS-8091: Install FPA module.

## Description

Relates to #8091 .

## Testing done
verified module was installed

## Screenshots
![image](https://user-images.githubusercontent.com/8375335/154741739-30973ae2-d122-4d0b-baaa-544784bae1a1.png)


## QA steps

As cms admin
1. verify FPA module is installed
2. view the permissions page and evaluate the new functionality
3. Then validate Acceptance Criteria from issue
- [ ]  FPA module is installed
- [ ] Permissions page is more user-friendly

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
